qa/automation/ScoringPortal.Tests.Scenarios/1. Access to Portal/1.1. Access to Career Portal.feature

```diff
@@ -1,6 +1,59 @@
-Feature: 1.1. Access to Career Portal
+Feature: 1. Signing in with Google account
 
-Scenario: 1.1_01 User can open Career Portal
-	Given 
-	When 
-	Then 
+Background:
+    Given User goes to https://qa.ddso-spot.quantori.com site
+    And User has Google account with credentials:
+    | email         | anastasiadmitrieva@gmail.com |
+    | phone number  | +375297773322                |
+    | password      | w763e!6&_N                   |
+          
+Scenario: 1.1.1  User opens email page when not authorized in Google account
+    Given User is on initial https://qa.ddso-spot.quantori.com page
+    When User clicks on "Sign in with Google" button
+    Then User is redirected to authorization form
+    
+Scenario: 1.1.2 User tries to sign in with email
+    Given User is on Google authorization page

```

Comment:
Please, consider rephrasing to 'Given User is on Quantori Scoring portal authorization page
And User clicks on "Sign in with Google" button
And User is redirected to authorization form
And Google authorization page is opened' so your Given would be consistent with your Background.